library/blivet.py

```diff
@@ -623,8 +623,13 @@ def _create(self):
         if self._device:
             return
 
-        if safe_mode:
-            raise BlivetAnsibleError("cannot create new RAID in safe mode")
+        for spec in self._volume["disks"]:
+            disk = self._blivet.devicetree.resolve_device(spec)
+            if not disk.isleaf or disk.format.type is not None:
```

Comment:
The check for `safe_mode` has to be a bit more careful, like the ones in `BlivetVolume._reformat` and `BlivetBase._manage_one_encryption`. I see that `BlivetPool._create_members` also needs to check for `device.original_format.name != get_format(None).name` (to catch formatting reported by blkid but not recognized/handled by blivet).

Reply:
Fixed.